src/apiclient.js

```diff
@@ -48,11 +48,10 @@ function ApiClient(token){
 		});
 	}
 
-	function _postMedia(type, payload, options){
+	function _postMedia(type, payload, options, api_method){
```

Comment:
The other args in the file use `camelCase`, so I would think this would be better if it's called `apiMethod`.